<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZipCodes::GenericZipCodesWidget</class>
 <widget class="QWidget" name="ZipCodes::GenericZipCodesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Generic ZipCodes database creator</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This plugins uses the &lt;a href=&quot;http://download.geonames.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;GeoNames database&lt;/span&gt;&lt;/a&gt; webservice to create a zipcodes database for the FreeMedForms project.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="downloadButton">
     <property name="text">
      <string>&amp;Download list of available countries</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="localeDataGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Zip code information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <widget class="QPushButton" name="startJob">
        <property name="text">
         <string>Create database package of this country</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="availCountriesLabel">
          <property name="text">
           <string>Available countries:</string>
          </property>
          <property name="buddy">
           <cstring>availCountriesCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Utils::CountryComboBox" name="availCountriesCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="countryCodeLabel">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This code is the official &lt;a href=&quot;http://en.wikipedia.org/wiki/ISO_3166-1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;ISO-3166-1 Alpha-2&lt;/span&gt;&lt;/a&gt; code&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;amp;Country code (to save as):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="buddy">
           <cstring>countryCodeLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="countryCodeLineEdit"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QTableView" name="zipsTableView">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::CountryComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">utils/widgets/countrycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>availCountriesCombo</tabstop>
  <tabstop>countryCodeLineEdit</tabstop>
  <tabstop>zipsTableView</tabstop>
  <tabstop>startJob</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
